platform/ios/Info.plist Keys.md

```diff
@@ -0,0 +1,15 @@
+# Info.plist Keys
+
+The SDK supports several custom `Info.plist` keys in your application in order to configure various settings. 
+
+## MGLMapboxAccessToken
+
+Use this key to set your Mapbox API [access token](https://www.mapbox.com/help/define-access-token/). This key is required for all map use. 
```

Comment:
This would be a good place to consolidate the explanation that’s currently in [`+[MGLAccessToken setAccessToken:]`](https://github.com/mapbox/mapbox-gl-native/blob/e7bfa7849e4ed7a80d27b63ac2f504660ecfb14b/platform/darwin/src/MGLAccountManager.h#L14-L23) and [MGLMapView](https://github.com/mapbox/mapbox-gl-native/blob/e7bfa7849e4ed7a80d27b63ac2f504660ecfb14b/platform/ios/src/MGLMapView.h#L69-L74). Actually, since a link back to this document won’t work in Quick Help, duplicating that explanation would be preferable.